### ISSUE TITLE
I am a wizard, I can float (though not on command)

### ISSUE THREAD

**Issue by Barerock** [tr7zw/FirstPersonModel#435]
![2024-03-10_23 18 29](https://github.com/tr7zw/FirstPersonModel/assets/38869993/7093e86d-7856-499f-b0c6-a647f36440e0)
My legs get stuck in this running animation in first person after a couple steps. In third person it triggers for seemingly random amounts of time at random, which makes it hard to take a picture of. I tried several times.

I just kinda only now noticed this somehow while placing dirt. I've been looking more or less straight forward for a WEEK while digging a hole, so idk how long it's been an issue. Legs are generally kinda visible though so I don't think it's older than today at least.

Here's my latest log.

[latest.log](https://github.com/tr7zw/FirstPersonModel/files/14553165/latest.log)

I have quite a few mods, but I don't believe any are ones that edit player animations except your two. I do have a speed enchantment on my boots, so maybe the animation is playing too fast and getting stuck? I'll test in another world real quick, but here ya go.

Our last interaction wasn't awesome, I want to say that I completely understand your stance and accept any and all decisions you make - not that this has any real impact on you personally. At the very least, you know I'm not trying to give you a hard time, I just like things to do things <3

**Comment by Barerock** [comment]
I can't get this to occur at all in a creative world that is a slightly older copy of the survival one in that screenshot. I've got the speed boots AND a potion on, and I tested with normal speed. Nada.

Maybe reloading fixed it.

**Comment by Barerock** [comment]
No, reloading didn't fix it. This is unique to the survival world.

**Comment by downloders** [comment]
Possible mod compatibility issue

**Comment by Barerock** [comment]
I can't recreate this in a side world in any realistic amount of time, so I'm limited in what I can do. I started by disabling 2 obvious character mods that I don't use. Neither affected the behavior.

If I continue disabling mods I will break my world, so this really isn't easy to pinpoint.

**Comment by Barerock** [comment]
I've more or less disabled every mod that interacts with the player itself or the inventory. The problem seems to occur under and above Y = ~70 and Y=75. Weirdly, going uphill temporarily dissipates the issue, when it appears in short fraction of a second bursts, but never to the extent of seeing my leg all the time.

I also only now noticed that the _entire_ walking and running animations system stops working. Neither my legs nor arms move and my body doesn't bob, only my camera.

Again, no idea what could possibly be doing this. I've disabled at least 24 mods by now and the only mods left after this batch are either very light content additions with few or no items or player interacting mechanics or full on performance mods. Things like Thin Air, Cold Sweat, Hearth and Home, and It Takes a Pillage are among these, as well as world gen overhauls with no new blocks, etc. The usual slew of performance mods like Embeddium, Ferrite Core, and many more. You can see my mod list in the latest log above.

**Comment by Barerock** [comment]
Oh, and I figured out that a full current copy of the world will allow me to recreate the issue in some areas. Problem is, I can't remove too many or the world no longer works and I can't reproduce the issue.

**Comment by Barerock** [comment]
It is 100% unrelated to chunk border alone. In the same chunk I am walking normally and abnormally, usually divided by a Y value, but not necessarily. It does seems to be more buggy in some chunks vs others, and I've noticed that it also relates to a weather bug I've had that makes some chunks not update their skybox color or even incorrectly adjust their skybox to rainy weather after none such events occurred.

So right now I'm testing each and every chunk loading mod I have on individually. Seems to me that this is a chunk update problem, not a FPM problem.

**Comment by Barerock** [comment]
Physics Mod Pro is the culprit. Among 196 mods (not my whole list), loading Not Enough Animations and Physics Mod Pro together cause this issue.

**Comment by tr7zw** [comment]
Huh, I had physics mod and NEA running together a lot without issues. To be fair, haven't played much with the full/pro version. I know that there was some mod that caused with NEA to sync the crouch between all players for whatever reason, but the walking animation freezing is weird. That would I think require that the player itself doesn't get ticked/the tick not getting noticed by NEA.

**Comment by Barerock** [comment]
Yeah, I have as well and am dismayed. I'm currently trying to load with the bare minimum requirements on that world, but it's looking like it's too baked to load without at least half the mods present. Instead, unfortunately, I think I will have to do an A-Z removal check to see if there's something else interacting with either mod.

**Comment by Barerock** [comment]
And a new world just doesn't have the bug right off the bat like I need.

**Comment by Barerock** [comment]
I'm now completely rid of all but the core mods that I couldn't remove without crashing. I'm 1 by 1 removing each, thoughwithout EVERY content addition mod besides Terralith, Tectonic, Continents, Nullscape, Physics Mod Pro, First Person Model, Player Animator Lib, and Not Enough Animations I still have the issue. This leads me to believe that this is a direct incompatibility between PMP and NEA.

**Comment by Barerock** [comment]
![image](https://github.com/tr7zw/FirstPersonModel/assets/38869993/3c95ba5e-83a2-4dfb-a86a-40af58bc7c73)
This is the final mod set I require to load the world. None of the extras mess with anything but world gen.
The issue persists.

**Comment by Barerock** [comment]
![2024-03-16_15 10 53](https://github.com/tr7zw/FirstPersonModel/assets/38869993/08546434-7c89-473e-be80-510f321b201a)
A 3rd person image of the issue for better clarity. The entire walking animation stops, including when sneaking.

**Comment by Barerock** [comment]
Further removing Continents and Tectonic does not resolve it. Blueprint cannot be removed, the world doesn't load.

**Comment by Barerock** [comment]
Found the issue.
![image](https://github.com/tr7zw/FirstPersonModel/assets/38869993/26e86660-5378-404d-a56e-aea6aa76d2fa)
This needs to be OFF

**Comment by downloders** [comment]
Congrats! bro

**Comment by tr7zw** [comment]
As I said, Animation Smoothing needs that the player knows when it gets ticked. So Physics mod pro breaks that somehow.
